doc/admin/installation.md

```diff
@@ -19,10 +19,6 @@ Moreover, the DAOS stack leverages the following open source projects:
 
 -   [*SPDK*](http://spdk.io/) for userspace NVMe device access and management.
 
--   [*FIO*](https://github.com/axboe/fio) for flexible testing of Linux I/O
```

Comment:
Isn't this still part of the build? I know we're removing the burnin capabilities for now but I thought that there were new tests that use it.

Reply:
Good point.